tests/unit/test_redis_task.py

```diff
@@ -22,7 +22,7 @@
 
 @pytest.fixture(autouse=True)
 def lower_ttl_for_test():
-    RedisTask.DEFAULT_TTL = 10  # Play nice with local infrastructure
+    RedisTask.DEFAULT_TTL = 1000  # Play nice with local infrastructure
```

Comment:
What's the reason for this change?  For example, I see no change to the `RedisTask` tests.

Reply:
Originally changed to 1000 to test expiry behavior in debugger, but didn't change back.